### PR TITLE
Added support for ATSAME54 and ATSAM4S16

### PR DESCRIPTION
**Pull request by john-terrell**
* Added support for ATSAME54 and ATSAM4S16 MCUs.
* Modified update.sh script to create XSL and PAC directories to simplify  adding new SVDs.
*

### CODE REVIEW

File: pac/atsam4s16b/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["John Terrell <john.terrell@outlook.com>"]
+categories = ["embedded", "hardware-support", "no-std"]
+description = "Peripheral access crate for the ATSAM4S16B microcontroller"
+keywords = ["arm", "cortex-m", "atsam4s16b", "svd2rust"]
+license = "MIT OR Apache-2.0"
+name = "atsam4s16b-pac"
+repository = "https://github.com/atsam4-rs/atsam-pac"
```
Comment (haata) [inline pac/atsam4s16b/Cargo.toml:8]: We'll probably need to adjust all the urls that point to github to say atsam-rs instead.

**Review by haata — CHANGES_REQUESTED** [review]

File: pac/atsam4s16b/Cargo.toml (hunk 1 above)
Comment (john-terrell) [reply pac/atsam4s16b/Cargo.toml:8]: Good call.  I will modify them.

Comment (john-terrell) [reply pac/atsam4s16b/Cargo.toml]: Good catch!

**Review by haata — APPROVED** [review]
:+1:

**Pull request merged**